src/components/VerifiedBilingualTable.tsx

```diff
@@ -19,23 +20,46 @@ import {
   Tooltip,
   Typography,
 } from "@mui/material";
-import { Controller, useFieldArray, useFormContext } from "react-hook-form";
+import {
+  Controller,
+  useFieldArray,
+  useFormContext,
+  useWatch,
+} from "react-hook-form";
 import { useTranslation } from "react-i18next";
 
-const VerifiedBilingualTable = ({ path }: { path: string }) => {
-  const { control, setValue, getValues } = useFormContext();
+const VerifiedBilingualTable = ({
+  path,
+  valueColumn = false,
+  unitOptions,
+}: {
+  path: string;
+  valueColumn?: boolean;
+  unitOptions?: string[];
+}) => {
```

Comment:
Define an interface instead of having it inline for readability / consistency with other functions?

Reply:
done